RELEASE.md

```diff
@@ -15,7 +15,7 @@ Release cadence of first pre-releases being cut is 6 weeks.
 | v0.45   | 2021-01-13                                 | Lili Cosic (GitHub: @lilic)                 |
 | v0.46   | 2021-02-24                                 | Sergiusz Urbaniak (GitHub: @s-urbaniak)     |
 | v0.47   | 2021-04-07                                 | Simon Pasquier (GitHub: @simonpasquier)     |
-| v0.48   | 2021-05-19                                 | **searching for volunteer**                 |
+| v0.48   | 2021-05-19                                 | Matthias Loibl (GitHub: @metalmatze)        |
 | v0.49   | 2021-06-30                                 | **searching for volunteer**                 |
```

Comment:
can we have someone signed up for the next release? @paulfantom?

Reply:
If nobody is available, I can volunteer

Reply:
I can take it ;)